src/components/test-title.tsx

```diff
@@ -7,16 +7,20 @@ const TestTitle = () => {
 
   return (
     <div>
-      <h2 style={{ textAlign: 'center', marginTop: '300px' }}>{store.hello}</h2>
-      <div
-        style={{
-          display: 'flex',
-          justifyContent: 'center',
-          columnGap: '30px',
-        }}
-      >
-        <button onClick={store.sayBye}>Say bye</button>
-        <button onClick={store.sayHello}>Say Hello</button>
+      <div>
+        <h2 style={{ textAlign: 'center', marginTop: '300px' }}>
```

Comment:
@NikitK-deriv 
i suggest using classNames instead of style, and .module.css (or .module.scss after we add sass) files for each component instead of .css files 🙂

Reply:
Totally agree with Maryia

Reply:
Resolved @maryia-binary @eduard-deriv please check 🙏